vm-setup/roles/v1aX_integration_test/templates/clusterctl.yaml

```diff
@@ -0,0 +1,28 @@
+# Image is overriden in provider/container_name format
+# list of providers
+# cluster.x-k8s.io/provider: bootstrap-kubeadm
+# cluster.x-k8s.io/provider: cluste
```

Comment:
leftover or typo?

Reply:
It is a documentation

Reply:
Sorry I mean, L3
`# cluster.x-k8s.io/provider: cluste`

Reply:
oh, will fix it
Thanks

Reply:
Thanks